app/src/main/res/values-sw600dp-port/dimens.xml

```diff
@@ -405,4 +405,10 @@
   <!-- AudioLanguageFragment -->
   <dimen name="audio_language_recycler_view_padding_bottom">132dp</dimen>
 
+  <!--  RevisionCardFragment-->
+  <dimen name="revision_card_fragment_layout_min_width" tools:ignore="MissingDefaultResource">512dp</dimen>
+  <dimen name="revision_card_fragment_padding_bottom">160dp</dimen>
+  <dimen name="revision_card_fragment_layout_text_margin_top">36dp</dimen>
```

Comment:
This will work but I guess it is not preferred to do this way. cc @rt4914 PTAL

Reply:
HAve consulted to sir and the required changes were made